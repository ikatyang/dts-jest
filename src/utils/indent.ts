import {repeat} from './repeat';

export const indent = (str: string, spaces: number) =>
  str.replace(/^/mg, repeat(' ', spaces));
