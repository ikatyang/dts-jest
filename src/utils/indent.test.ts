import { indent } from './indent';

it('should return correctly', () => {
  expect(indent('line 1\nline 2\nline 3', 2)).toMatchSnapshot();
});
