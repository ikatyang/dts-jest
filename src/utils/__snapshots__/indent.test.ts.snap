// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should return correctly 1`] = `
"  line 1
  line 2
  line 3"
`;
